ocs_ci/utility/users.py

```diff
@@ -64,6 +64,41 @@ def delete_htpasswd_secret():
     exec_cmd(cmd)
 
 
+def delete_user(user_name=None):
+    """
+    Delete user
+
+    Args:
+        user_name (str): the user to delete
+    """
+    if user_name is None:
+        cmd = "oc delete user --all"
+    else:
+        cmd = f"oc delete user {user_name}"
+    try:
+        exec_cmd(cmd)
+    except Exception as e:
+        logging.error(e)
+
+
+def delete_identity(user_name=None):
+    """
+    Delete identity
+
+    Args:
+        user_name (str): the user to delete
+
+    """
+    if user_name is None:
+        cmd = "oc delete identity --all"
+    else:
+        cmd = f"oc delete identity my_htpasswd_provider:{user_name}"
+    try:
+        exec_cmd(cmd)
+    except Exception as e:
+        logging.error(e)
```

Comment:
Tox fails because logging is used directly (https://github.com/red-hat-storage/ocs-ci/actions/runs/3924241160/jobs/6708369701).

Reply:
done